samples/ResourceOwnerPasswordFlow/DeveloperNotes/ConformanceProfiles/fiddler-request-OP-Response-Missing.txt

```diff
@@ -0,0 +1,7 @@
+﻿POST http://localhost:50000/connect/token HTTP/1.1
```

Comment:
I like the output of Fiddler. For instance, it has Web View that makes errors easy to read, and has JSON view for making the JWT properties easy to copy/paste. It's pretty easy to use Fiddler for me. I'm not sure how I would structure the output of a console app to make it as helpful as Fiddler is for prototyping and troubleshooting. Once the prototyping is done, I then tend to create a simple web user interface like the one that's there now instead of moving into a console app. Thoughts?